online_test/settings.py

```diff
@@ -187,3 +188,6 @@
 SOCIAL_AUTH_FACEBOOK_PROFILE_EXTRA_PARAMS = {
     'fields': 'id, name, email'
 }
+
+GOOGLE_RECAPTCHA_SECRET_KEY = 'Your Key here'
+data_key = 'Google API key here'
```

Comment:
Follow any one method while naming a variable either keep the variable names in upper case or lower case.
Don't use lower case here since we have already used the upper case for other variables